plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.ts

```diff
@@ -54,8 +55,21 @@ export class GkeClusterLocator implements KubernetesClustersSupplier {
         return { key: mrl.getString('key'), value: mrl.getString('value') };
       }) ?? [];
 
+    let storeAuthProviderString: string;
+    let getGkeProperty;
+    try {
+      getGkeProperty = config.getString('authProvider');
```

Comment:
Could this try catch be

`config.GetStringOptional('authProvider') ?? "google"`

Reply:
I would use the version suggested by @jamieklassen below which aims to resolve the same thing

Reply:
@mclarke47 Done!

Thanks